packages/performance-tests/test/GitUtils.ts

```diff
@@ -0,0 +1,12 @@
+import util = require("node:util");
```

Comment:
don't use `import blah = require()`, I know typescript suggests it sometimes but use either:
```ts
import fs from "fs";
//or
import * as fs from "fs";
```
whichever is fine, depends on the tsconfig settings if it's allowed.

Reply:
done